Sources/UInt128.swift

```diff
@@ -648,7 +648,7 @@ extension UInt128 : Comparable {
 
 // MARK: - ExpressibleByStringLiteral Conformance
 
-extension UInt128 : ExpressibleByStringLiteral {
+extension UInt128 : ExpressibleByStringLiteral, LosslessStringConvertible {
```

Comment:
I thought that one of the protocols I declare support for included LosslessStringConvertible. I'll take a look again on Wednesday to make sure.